src/nat/builder/function.py

```
@@ -497,15 +576,64 @@ def get_included_functions(self) -> dict[str, Function]:
         missing = set(self._config.include) - set(self._functions.keys())
         if missing:
             raise ValueError(f"Unknown included functions: {sorted(missing)}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)